src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -1024,6 +1028,13 @@ public void ensureFullCommit() {
         db.ensureFullCommit();
     }
 
+    /**
+     * Database URI helper.
+     */
```

Comment:
I noticed this failed in my local build because there is no `@return` in the javadoc. I have updated Travis to make sure these errors get picked up too.
```
src/main/java/com/cloudant/client/api/Database.java:1034: warning: no @return
    public DatabaseURIHelper getDbUriHelper() {
```